Automattic/wp-calypso
# Woo Express: Remove refs to plans/wooexpress-small and plans/wooexpress-medium feature flags

Related to #75560

## Proposed Changes

* Remove references in the code to the feature flags `plans/wooexpress-small` and `plans/wooexpress-medium`

## Testing Instructions



* Apply this patch
* Make sure the `/plans` page still functions as expected for all Woo Express plan types (Trial, Essential, Performance) as well as for expired sites.

## Pre-merge Checklist



- N/A [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- N/A Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- N/A Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- N/A For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
